### ISSUE TITLE
Remove Eye icon, Change QR scan icon

### ISSUE THREAD

**Issue by designsats** [GaloyMoney/galoy-mobile#2803]
1. Eye icon to hide balance is redundant
![image](https://github.com/GaloyMoney/galoy-mobile/assets/17384787/94333bf9-974f-4326-9f3f-0d251bd006c1)

3. Scan QR icon should be changed everywhere applicable. 
Current icon is industry standard for "Show QR" not scan. 

![image](https://github.com/GaloyMoney/galoy-mobile/assets/17384787/b4522e22-43de-49e5-9736-50761d19ecdd)

**Comment by MaxwellDG** [comment]
1. While the eye button is redundant, I can see it's purpose. Took me quite a while to discover that tapping the portfolio balance did the same thing. I'd leave the icon there, or if you really want we could move it elsewhere? Faded and to the right of the chart icon in the top left maybe?

**Comment by designsats** [comment]
> @designsats While the eye button is redundant, I can see it's purpose. Took me quite a while to discover that tapping the portfolio balance did the same thing. I'd leave the icon there, or if you really want we could move it elsewhere? Faded and to the right of the chart icon in the top left maybe?

Yes - you are right, its not intuitive enough without the eye. Lets keep the icon.

**Comment by nicolasburtey** [comment]
I need a new SVG to replace th existing QR code SVG: https://github.com/GaloyMoney/galoy-mobile/blob/main/app/assets/icons-redesign/qr-code.svg

**Comment by designsats** [comment]
![qr-scanner](https://github.com/GaloyMoney/galoy-mobile/assets/17384787/fd5030ba-b75e-4fbb-850a-7dccce113239)
